examples/js/loaders/SVGLoader.js

```diff
@@ -1092,144 +1118,158 @@ THREE.SVGLoader.prototype = Object.assign( Object.create( THREE.Loader.prototype
 
 		function parseNodeTransform( node ) {
 
-			var transform = new THREE.Matrix3();
+			var transform = new Matrix3();
```

Comment:
Does this work?

Reply:
My mistake. Fixed.

Reply:
Hmm, are you sure it's fixed?

Reply:
Sorry, I must have failed to git add it.